lib/Mojolicious/Guides/Growing.pod

```diff
@@ -752,9 +734,7 @@ C<@INC>, allowing us to start the application from outside its home directory.
 
   #!/usr/bin/env perl
 
-  use strict;
-  use warnings;
-
+  use Mojo::Base -strict;
```

Comment:
This is wrong. Now this example is inconsistent with  `mojo generate app`.

Reply:
reverted